src/gl/align-gl.cpp

```diff
@@ -274,8 +274,15 @@ align_gl::align_gl(rs2_stream to_stream) : align(to_stream, "Align (GLSL)")
 
 align_gl::~align_gl()
 {
-    perform_gl_action([&]()
+    try
     {
-        cleanup_gpu_resources();
-    }, []{});
+        perform_gl_action([&]()
+        {
+            cleanup_gpu_resources();
+        }, [] {});
+    }
+    catch (...)
+    {
+        LOG_INFO("Exception occurred during align_gl destruction");
```

Comment:
General comment regarding all changes
1. Debug level log is enough, no point of showing these errors to the user.
2. Log message should state the action that went wrong, not the location. e.g. "Exception occurred during align_gl destruction" would better be "Error while cleaning up gpu resources"

Reply:
Fixed according to 1 and 2.